excalidraw/src/worker.js

```diff
@@ -20,6 +21,12 @@ export default class Worker {
     try {
       await page.setViewport({ height: 800, width: 600 })
       await page.goto(this.pageUrl)
+      await page.addScriptTag({
+        path: `${path.join(__dirname, '..', 'assets', 'excalidraw', 'excalidraw.production.min.js')}`
+      });
```

Comment:
why do we need to add it dynamically?

Reply:
I tried to leave it static with the index.html, but then it did not work and the "window.EXCALIDRAW_ASSET_PATH" was not set correctly and a "TimeOutError" occurred again.
It is irrelevant whether "window.EXCALIDRAW_ASSET_PATH" is defined before or after "await page.goto(this.pageUrl)", in both cases a "TimeOutError" occurs.
It worked with the dynamic loading.
Unfortunately, I am not familiar enough with the code and the programming language to really say why this is so.

Reply:
Hmmm 🤔
Thanks for your reply. I will test locally to try to understand why it's happening.

Reply:
I can confirm that we need to set `window.EXCALIDRAW_ASSET_PATH` before loading `excalidraw.production.min.js` since this JavaScript file will use the value of `window.EXCALIDRAW_ASSET_PATH`.